cmd/describe/cluster/cmd.go

```diff
@@ -335,6 +336,11 @@ func run(cmd *cobra.Command, argv []string) {
 			scheduledUpgrade.NextRun().Format("2006-01-02 15:04 MST"),
 		)
 	}
+	str = fmt.Sprintf("%s"+
+		"Disable workload Monitoring: %t\n",
```

Comment:
Do we want to show this always or only when it's disabled? 
Also, can we change the output to `User Workload Monitoring: disabled|enabled`

Reply:
In OCM we show it always. Changed it to show only when disabled just to avoid more irrelevant information in cli.